include/seqan3/io/sequence/sequence_file_format.hpp

```diff
@@ -31,19 +31,52 @@
 // DAMAGE.
 //
 // ============================================================================
-// Author: Joerg Winkler <j.winkler AT fu-berlin.de>
-// ============================================================================
+
+/*!\file seqan3/io/sequence/sequence_file_format.hpp
+ * \ingroup io
+ * \author Joerg Winkler <j.winkler AT fu-berlin.de>
+ * \brief Defines the concept of sequence file formats.
+ */
 
 #pragma once
 
-#include <seqan3/alphabet/nucleotide/dna4_sequence.hpp>
-#include <seqan3/io/sequence/sequence_file_in.hpp>
 #include <string>
 #include <fstream>
 
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/io/sequence/sequence_file_in.hpp>
+
 namespace seqan3
 {
-
+/*!\interface seqan3::sequence_file_format_concept
+ * \brief The generic concept for sequence file formats.
+ * \relates sequence_file_in_traits_concept
+ * \tparam t The sequence file format type.
+ * \ingroup io
+ * \details
+ * The requirements for this concept are given as related functions and metafunctions.
+ * Types that satisfy this concept are shown as "implementing this interface".
+ */
+/*!\fn void read(dna4_string sequence, std::string meta, std::string quality, std::ifstream stream, options_type opt)
```

Comment:
this functions seems to miss a place where to life.

try `\ingroup io`

Reply:
`read` and `write` seems to be global functions, but your concept below assumes class functions.